### PR TITLE
Cleaner headers

### PR DESCRIPTION
**Pull request by shaundillon**
Combines embedded and non-embedded headers and instead introduces the idea of Editing and Browsing headers.

picture 142
picture 143
picture 144
picture 145

### CODE REVIEW

**Comment by clloyd** [comment]
Possible to get some screen shots?

**Comment by shaundillon** [comment]
@clloyd You can, but you'll be disappointed. They look the same haha

File: public/styles/base/_typography.scss [hunk 1]
```diff
@@ -6,26 +6,26 @@ body {
...
-h1, h2, h3, h4, h5, h6, .preview__title, .preview__score {
+h1, .h1, h2, .h2, h3, .h3, h4, .h4, h5, .h5, h6, .h6, .preview__title, .preview__score {
   font-family: $title-font-stack;
   font-weight: normal;
   font-style: normal;
   margin: 0;
 }
 
-h1 {
+h1, .h1 {
```
Comment (susiecoleman) [inline public/styles/base/_typography.scss]: Why have the classes been added? (I'm not say it's wrong I'm just curious)

File: public/js/components/Header/EditHeader.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+  isEditor = () => {
+    return location.pathname.search(new RegExp('\/atoms\/.*\/.*\/edit', 'g')) >= 0;
+  }
+
+  timeSinceLastModified = () => {
+    if (this.props.atom.contentChangeDetails.created || this.props.atom.contentChangeDetails.lastModified) {
+      const dateNow = Date.now();
+      const lastModified = this.props.atom.contentChangeDetails.lastModified ? this.props.atom.contentChangeDetails.lastModified.date : this.props.atom.contentChangeDetails.created.date;
+      return distanceInWords(dateNow, lastModified, {addSuffix: true});
```
Comment (susiecoleman) [inline public/js/components/Header/EditHeader.js]: What does distanceInWords do?

**Pull request merged**